redesign/about.html

```diff
@@ -0,0 +1,183 @@
+      </div>
+      <div>
+        <p class="pl3 b dib mv1">Origin:</p>
+        <p class="dib pl3 i mv0 times-new-roman">Do What You Love</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1 dib">1.</p>
+        <p class="dib pl2 i mv0 times-new-roman w-80 w-90-ns v-top pt1">
+          Community of people on a mission to change the world using technology.
+        </p>
+      </div>
+
+    </section>
+
+    <section class="dwyl-bg-dark-gray white pt5 pb4 pb5-ns ph6-ns ph4 center">
+      <h2 class="f2 mv0">More Than Just Software</h2>
+      <p class="istok-web">We solve real problems with technology. We are different by design.</p>
+      <button class="bn dib link mb3 ph3 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4 pointer istok-web">Join Our Community</button>
+      <button class="bn dib link ml2 mb3 ph3 pv2 dwyl-bg-mint white br2 f5 shadow-4 pointer istok-web">Meet the Team</button>
+
+      <h2 class="f2 mt5 mb2">We believe in:</h2>
+      <div class="dib w-45-ns">
+        <h3 class="dwyl-yellow mt1">People 🤝</h3>
+        <p class="lh-copy">
+          Our people are our primary focus, not profit. We have a
+          <a href="#" class="dwyl-yellow">worldwide community of over 350 creative technologists.</a>
+          We believe in inclusivity and accessibility and want to enrich the
+          lives of everyone in our community. That’s why we made
+          <a href="https://www.focushub.co.uk/" class"dwyl-yellow">a home for our London-based team at Focus Hub.</a>
+        </p>
+      </div>
+      <div class="dib w-45-ns ml4-m ml5-ns v-top">
+        <h3 class="dwyl-yellow mt1">Transparency 🔎</h3>
+        <p class="lh-copy">
+          Open-Source and Open-Kimono: we’re building dwyl for all to see! The
+          vast majority of dwyl projects are written open-source. This allows
+          code modules to improve and adapt over time allowing for a more robust
+          and up-to-date application.
+        </p>
+      </div>
+      <div class="dib w-45-ns">
+        <h3 class="dwyl-yellow">Looking After the Planet 🌍</h3>
+        <p class="lh-copy">
+          We recycle and plan to be carbon neutral by 2018, but it’s not just
+          the planet’s environment we want to improve. We love projects that
+          have a positive effect on the world. Take a look at
+          <a href="/portfolio" class="dwyl-yellow">the kinds of project we love.</a>
+        </p>
+      </div>
+      <div class="dib w-45-ns ml4-m ml5-ns v-top">
+        <h3 class="dwyl-yellow">Organic Growth 🌱</h3>
+        <p class="lh-copy mb1">
+          We are growing organically, without investment, to be flexible enough
+          to move in the direction our community takes us! Want to be a part of our growth.
+        </p>
+        <a href="#" class="dwyl-yellow">Sound like somewhere you’d like to work?</a>
+      </div>
+
+    </section>
+    <section class="dwyl-bg-mint center pt4-ns pb5-ns">
```

Comment:
It would be great to get a nice big comment here (and in most sections) so we don't have to go section hunting, e.g.
```html
<!-- ------------------------------
              CONTACT FORM
-------------------------------- -->
```